docs/python/1.6/python-appapi-devguide-5.md

```diff
@@ -66,7 +66,7 @@ def main():
     kelvin = source.parallel(4).map(temperature_sensor_functions.convertToKelvin)
     end = kelvin.end_parallel()
     end.sink(print)
-          streamsx.topology.context.submit("STANDALONE", topo.graph)
+    streamsx.topology.context.submit("STANDALONE", topo.graph)
```

Comment:
If I search for `topo.graph` in this repo, I see 12 other instances in the following files:

- docs/python/1.4/python-appapi-devguide-3.md
- docs/python/1.4/python-appapi-devguide-4.md
- docs/python/1.4/python-appapi-devguide-5.md
- docs/4.2/qse-getting-started.markdown

Shall I fix those as well in this PR?